_health-care/_js/_components/_form-elements/ErrorableInput.jsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import _ from 'lodash';
+
+/**
+ * A form input with a label that can display error messages.
+ *
+ * Validation has the following props.
+ * `errorMessage` - Error string to display in the component.
+ *                  When defined, indicates input has a validation error.
+ * `label` - String for the input field label.
+ * `placeholder` - placeholder string for input field.
+ * `required` - boolean. Render marker indicating field is required.
+ * `value` - string. Value of the input field.
+ * `onValueChange` - a function with this prototype: (newValue)
+ */
+class ErrorableInput extends React.Component {
+  constructor() {
+    super();
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  componentWillMount() {
+    this.inputId = _.uniqueId('validated-input-');
```

Comment:
Maybe we can match the name of the component for clarity and call this `errorable-input-`?